app/src/main/res/layout/activity_main.xml

```diff
@@ -4,14 +4,14 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="#3D5AF8"
```

Comment:
Please move this color to the styles or colors file and give it some name

Reply:
done